src/pages/ProfilePage.js

```diff
@@ -121,7 +114,7 @@ function ProfilePage(props) {
     const phoneNumber = getPhoneNumber(details);
     const phoneOrEmail = isSMSLogin ? getPhoneNumber(details) : login;
 
-    const isCurrentUser = _.keys(props.loginList).includes(login);
+    const isCurrentUser = props.session.accountID === accountID;
```

Comment:
Use this same fix for `<DetailsPage />`

Reply:
Done but the route param is email so the comparison is done with session.email.